docs/docs/coverage/language/index.md

```diff
@@ -1,20 +1,21 @@
 # Programming Language
 
-Trivy supports programming languages for 
+Trivy scans programming language packages in the following scanners: 
 
 - [SBOM][sbom]
 - [Vulnerabilities][vuln]
 - [Licenses][license]
 
-## Supported languages
-The files analyzed vary depending on the target.
-This is because Trivy primarily categorizes targets into two groups:
+## Target types
+Trivy categorizes targets into either Pre-build and Post-build targets. The files analyzed vary depending on the target type:   
+
+- **Pre-build** is meant for scanning code projects, where packages are likely in package manager lock files (e.g `package-lock.json`). 
+- **Post-build** is meant for scanning deployable artifacts (e.g vm, container) where packages are likely "installed" (e.g in `node_modules`) and source code (including lock files) is not available.
```

Comment:
I think we can just add link to terminology - https://github.com/aquasecurity/trivy/blob/2972477fd25e5aeec873422d7976660451921380/docs/docs/references/terminology.md?plain=1#L32-L36
wdyt?

Reply:
I think the main docs should cover all the relevant information, and the terminology page is the optional one (collecting in one place all the common terms). If anything, I would link from the terminology page to the main doc instead.

Reply:
https://github.com/aquasecurity/trivy/pull/8305/commits/7fc4ec7d185bb5b599053ba9b22583e1c4589ba9

Reply:
This also suits me.
My goal was to have the information in one place and to correct/update it (if necessary) would also only need to be in one place.